image_label.py

```diff
@@ -34,17 +32,17 @@ def load_labels(filename):
     parser.add_argument(
         '-i',
         '--image',
-        default=os.getenv('SNAP')+'/grace_hopper.bmp',
+        default=os.getenv('SNAP', '.') + '/grace_hopper.bmp',
```

Comment:
I suggest keeping packaging logic (SNAP var) outside the applications. The default image, model, label file paths can be kept relative, and the base path can be passed to it as an environment variable in snapcraft.yaml.

```suggestion
        default=os.getenv('WORKDIR', '.') + '/grace_hopper.bmp',
```